modules/installing-oci-about-agent-based-installer.adoc

```diff
@@ -8,9 +8,6 @@
 
 You can install an {product-title} cluster on {oci-first} by using the Agent-based Installer. Both Red Hat and Oracle test, validate, and support running {oci} and {ocvs-first} workloads in an {product-title} cluster on {oci}.
 
-:FeatureName: Using the Agent-based Installer to install an {product-title} cluster on {oci} that is configured with a virtual machine (VM) compute instance
```

Comment:
Just to confirm that 4.14 will no longer be TP for the VM route? This would mean that the 4.14 RN TP table needs an update to state GA?

Reply:
Yes